_posts/2016-09-27-clean.markdown

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Clean
+date: 2016-09-27 
```

Comment:
Thanks for PR! For the sake of common formatting, would You please be so kind and add a timestamp to this date? If You don't know the exact time then please set it as 00:00:00.

Thanks!